docs/components/similaritygraph.rst

```diff
@@ -24,7 +24,7 @@ Example
 
 .. raw:: html
 
-    <svg id="similaritygraph-example" width="700" height="700"></svg>
+    <div id="similaritygraph-example" width="700" height="700"></svg>
```

Comment:
You are opening with a div but still closing with an svg. https://xkcd.com/1144/